src/vm/frames.h

```diff
@@ -2535,6 +2535,10 @@ class GCFrame : public Frame
 
     // Keep as last entry in class
     DEFINE_VTABLE_GETTER_AND_DTOR(GCFrame)
+
+#ifndef FEATURE_PAL
```

Comment:
Why not use it everywhere?

Reply:
PAL already has a destructor that does the same thing - all frames in PAL do.

Reply:
I meant - with FEATURE_PAL.